New blog, updated readme and maintainers files

microwavables opened the pull request:
Added new blog post, images along with post, added community info to README, updated MAINTAINERS file

## What this PR does / why we need it
<!--
Updated maintainers doc to reflect new maintainers, stakeholders, and other changes. Added community section on README for folks to get more info about how to join our community. Added new blog post written by Cora Iberkleid.
-->

## Which issue(s) this PR fixes
<!--
None.
-->
Fixes: #

## Describe testing done for PR
<!--
None
-->

## Special notes for your reviewer
<!--
blah
-->

Commit:
updated blogpost and added new image

updated blogpost and added new image

fixed linting

more fixing of the lint

New blog, updated readme and maintainers files

Added new blog post, images along with post, added community info to README, updated MAINTAINERS file

Trying to fix linting issues

Some linting issues...

Commit:
updated blogpost and added new image

updated blogpost and added new image

fixed linting

more fixing of the lint

New blog, updated readme and maintainers files

Added new blog post, images along with post, added community info to README, updated MAINTAINERS file

Trying to fix linting issues

Some linting issues...

changed teaser image for blog

updated teaser image

stmcginnis commented on README.md:
```diff
@@ -134,3 +134,17 @@ Please submit [bugs or enhancements requests](https://github.com/vmware-tanzu/co
+
+## Join the Community and Make Tanzu Community Edition Better
+
+Tanzu Community Edition Better is better because of our contributors and maintainers. It is because of you that we can bring great software to the community. Please join us during our online community meetings. Details can be found on our [website](https://tanzucommunityedition.io/community/).
```
Looks like an extra "Better" in there? :)

microwavables replied:
SO MUCH BETTER! good catch :)

stmcginnis commented on README.md:
```diff
@@ -134,3 +134,17 @@ Please submit [bugs or enhancements requests](https://github.com/vmware-tanzu/co
+
+You can chat with us on Kubernetes Slack in the [#tanzu-community-edition channel](https://kubernetes.slack.com/archives/C02GY94A8KT) and follow us on Twitter at [@VMwareTCE](https://twitter.com/VMwareTCE).
+
+Note: If aren’t already a member on the Kubernetes Slack workspace, please first [request an invitation](https://slack.k8s.io/) to gain access.
```
s/If aren't/If you aren't/

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
```diff
@@ -0,0 +1,694 @@
+
+The next one is a management cluster (tce-mgmt-cluster), which is useful for use cases where you need to provision many clusters. I’ve used this management cluster, in fact, to provision the third “workload” cluster that you see here (tce-workload-cluster-1). These happen to be running on AWS.
+
+You can see by the color of the output that my context is pointing to the workload cluster. This is the one I will be using for this demo.
```
Hmm, probably should have a screenshot if the color matters, since this code block isn't going to have any special color formatting.

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
```diff
@@ -0,0 +1,694 @@
+
+You can see some namespaces with tkg in the name. This stands for Tanzu Kubernetes Grid and these namespaces contain resources related to core functionality of Tanzu Community Edition, like provisioning clusters, for example.
+
+There is also a namespace called tanzu-package-repo-global—that's where the package repository that I mentioned is installed.
```
s/tanzu-package-repo-global—that's/tanzu-package-repo-global - that's/

maybe?

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
```diff
@@ -0,0 +1,694 @@
+
+Since Harbor is already installed, we can go ahead and open the UI. We’ll just log in as admin, and let’s go ahead and create a new “demo” project to store the images we’re going to publish.
+
+!["Create a new demo project dashboard"](/img/deploy-app-post-img1.png)
```
I believe this needs to be `../img` to correctly reference the image path.

microwavables replied:
I'm confused, the images are showing up fine in the deploy preview to the site? :/

stmcginnis replied:
Chatted in slack - I think this was just my confusion from other similar changes recently that were actually referencing between the versioned docs and the unversioned blogs. This looks like it's actually right, so please ignore me. :)

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
```diff
@@ -0,0 +1,694 @@
+
+If we check back on Harbor, we can see these two images now on our registry. One of these is the build image and the other is the run image.
+
+!["Build and run images shown on dashboard"](/img/deploy-app-post-img2.png)
```
`../img/`

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
```diff
@@ -0,0 +1,694 @@
+
+We can check Harbor again, and we see two more images here. One has the set of Java buildpacks, and the other has the set of Go buildpacks.
+
+!["Java and Go buildpacks images shown on dashboard"](/img/deploy-app-post-img3.png)
```
`../`

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
```diff
@@ -0,0 +1,694 @@
+
+In this case the kpack CLI is only creating the ClusterBuilder resource in Kubernetes. You can see from the output it’s not publishing anything to Harbor. Instead, kpack itself is going to use this configuration and generate a new image for us. That’s happening on the cluster. We can hop over to Harbor and make sure the image is there.
+
+!["Generating new image shown on demo dashboard"](/img/deploy-app-post-img4.png)
+
+!["Generating new image shown on builder dashboard"](/img/deploy-app-post-img5.png)
+
```
`../`

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
````diff
@@ -0,0 +1,694 @@
+hello-world    True     CONFIG           harbor.tanzu.coraiberkleid.site/demo/hello-world@sha256:32c0ffc57c0c560913e24e6922d1f343a504b67f54ec1156ddcac38031180fd3    default
+```
+
+!["kpack is tagging with a build number and timestamp dashboard"](/img/deploy-app-post-img6.png)
````
`../`

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
```diff
@@ -0,0 +1,694 @@
+
+And you can see kpack is also tagging it with a build number and a timestamp.
+
+!["See what buildpacks were used, what run image is dashboard"](/img/deploy-app-post-img7.png)
```
`../`

stmcginnis commented on docs/site/content/posts/08-build-deploy-apps-w-tce.md:
````diff
@@ -0,0 +1,694 @@
+2        SUCCESS    harbor.tanzu.coraiberkleid.site/demo/hello-world@sha256:0313de8f942fa8aaee8cfc7997428a939d4523ab23d49d590aeedc7020b492fb    COMMIT    hello-world
+```
+
+!["hello-world on dashboard"](/img/deploy-app-post-img8.png)
````
`../`

Commit:
resolved issues

fixed some typos

Commit:
Merge branch 'build-deploy-blog' of https://github.com/microwavables/community-edition into build-deploy-blog

Commit:
accidentally reverted

I may have messed this up..

re-did some things to fix linting

please forgive me for all these commits, I messed up.

stmcginnis reviewed (approved):
Thanks for addressing comments - looks good to me.
